src/Wallet.purs

```diff
@@ -67,10 +70,21 @@ mkGeroWalletAff = Gero <$> mkCip30WalletAff "Gero" _enableGero
 
 foreign import _enableGero :: Effect (Promise Cip30Connection)
 
+isFlintAvailable :: Effect Boolean
+isFlintAvailable = _isFlintAvailable
+
+foreign import _isFlintAvailable :: Effect Boolean
+
+mkFlintWalletAff :: Aff Wallet
+mkFlintWalletAff = Flint <$> mkCip30WalletAff "Flint" _enableFlint
+
+foreign import _enableFlint :: Effect (Promise Cip30Connection)
+
 cip30Wallet :: Wallet -> Maybe Cip30Wallet
 cip30Wallet = case _ of
   Nami c30 -> Just c30
   Gero c30 -> Just c30
+  Flint c30 -> Just c30
   _ -> Nothing
```

Comment:
Come to think of it, we should not use wildcard pattern here to make it harder to forget to add a new wallet when we do it.